amber-slsa-buildtype.json

```diff
@@ -0,0 +1,137 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "Describes a SLSA buildType, as defined in https://slsa.dev/provenance/v0.2.",
+  "type": "object",
+  "required": [
+    "buildType",
+    "buildConfig",
+    "materials"
+  ],
+  "not": {
+    "required": [
+      "invocation"
+    ]
+  },
+  "properties": {
+    "buildType": {
+      "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype.json"
+    },
+    "buildConfig": {
+      "type": "object",
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "Command passed to the builder image.",
+          "examples": [
+            "./scripts/runner build-functions-server"
+          ]
+        },
+        "outputPath": {
+          "type": "string",
+          "description": "The path, relative to the root of the git repository, where the artifact built by the command is expected to be found.",
+          "examples": [
+            "./oak_functions/loader/bin/oak_functions_loader"
+          ]
+        },
+        "expectedArtifactDigest": {
+          "type": "object",
+          "description": "Expected hash of the output artifact. Can be ommitted.",
+          "properties": {
+            "sha256": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "sha256"
+          ],
+          "additionalProperties": false,
```

Comment:
Why is it false? I think it is fine for this version, but should we keep it for future versions too?

Reply:
This goes back to my answer on https://github.com/project-oak/transparent-release/pull/3#discussion_r794467352. Our tooling can only realistically support a limited number of hashing functions.

We could allow other ones to be set as additional values, though we wouldn't be able to validate them. Perhaps that's fine though. :)

Reply:
Yes. I think it is ok for the schema to allow other ones, as long as it requires `sha256`. We only validate `sha256` for now.